### PR TITLE
Enabled dark mode

### PR DESCRIPTION
**Pull request by hs-07**
This commit enables **Dark Mode** on the site.
Changes I made:
- Inserted a toggle in Navbar called "ColorModeToggle"
- Made necessary style change in "style.module.css"
- Updated "docasaurus.config.js" file.

### CODE REVIEW

**Comment by dwbrown2** [comment]
Love the product experience!! Waiting for eng review.

**Comment by hs-07** [comment]
> This looks good! The toggle works well, however some of the logos look a little weird when swapping to dark mode. Any chance we have some alternatives @dwbrown2 ?

Thank you @jjarrett21 for the compliment😊
And yaa, some logos like CNCF and opencost (in navbar), looks a bit inappropriate when we switch to dark mode.

**Comment by hs-07** [comment]
> I agree they don't look perfect, but I'm ok for us to take that as a TODO if you want to merge this beforehand!

I have a solution.
Give me the list of the logos whose color are not perfect in dark mode. I will change them when we swap.😊🙌will that be ok?

**Comment by hs-07** [comment]
Yaa, this will be fine.

File: src/components/Partners.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ export default function Partners() {
   return (
     <div
       className="px-10 py-20 md:py-32 mt-20 md:mt-60"
-      style={{ backgroundColor: "#e5e5e5" }}
+      // style={{ backgroundColor: "#e5e5e5" }}
```
[the remark this answers is not in the document]
Comment (dwbrown2) [reply src/components/Partners.js:7]: This feels like the only outstanding comments and then we can merge!

**Comment by hs-07** [comment]
@jjarrett21 @dwbrown2 
Any update regarding merging the PR?

**Review by dwbrown2 — APPROVED** [review]

**Comment by dwbrown2** [comment]
Approved and ready to merge whenever you guys are!

**Comment by AjayTripathy** [comment]
THank you @hs-07 !

**Pull request merged**